doc/sphinx/addendum/extraction.rst

```diff
@@ -180,11 +180,14 @@ and commands:
 
 .. flag:: Extraction AutoInline
 
-   Default is on. The extraction mechanism inlines the :term:`bodies <body>` of
+   Default is off. When enabled, the extraction mechanism inlines the :term:`bodies <body>` of
    some defined :term:`constants <constant>`, according to some heuristics
    like size of bodies, uselessness of some arguments, etc.
-   Those heuristics are not always perfect; if you want to disable
-   this feature, turn this :term:`flag` off.
+
+   Even when this flag is off, recursors, projections, and a few
```

Comment:
By "recursor", you mean something a recursive definition, function, ...?  The meaning is not obvious, nor is it defined in the doc.  Can you substitute a short phrase for it?